ion/src/device/regs/cm4.h

```diff
@@ -40,14 +40,39 @@ class CM4 {
     REGS_BOOL_FIELD(SLEEPDEEP, 2);
   };
 
+  class STCSR : public Register32 {
```

Comment:
I'd rename this to `SYST_CSR` as the ARM ARM calls it. I know, even ARM has two different names for this register, but the one with the most doc is `SYST_CSR`. Same goes for `SYST_RVR` and `SYST_CVR`.